src/modules/admin/AdminRouter.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
Admin router is not connected. Please add AdminRouter to Main.tsx with path={`${match.path}/admin`}. Exact removal will be needed probably because of nested routing